data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/schema/BeaconHeadResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.schema;
+
+public class BeaconHeadResponse {
+  public final long slot;
```

Comment:
slot here but best_slot on the builder

Reply:
fair call, i'll rename it. I was kind of more interested in whether I should be doing a builder at all, but all feedback welcome.